monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional, List, Dict
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
+from monitoring.uss_qualifier.common_data_definitions import Severity
+
+class Result(ImplicitDict):
```

Comment:
Is this used anywhere?

Reply:
I was thinking this can be used in the simulator to generate / hold results, but I will remove it.